protocol-designer/src/organisms/WellOrderModal/WellOrderVisualization.tsx

```diff
@@ -0,0 +1,42 @@
+import b2t_l2r from '../../assets/images/well-order_b2t-l2r.jpg'
+import b2t_r2l from '../../assets/images/well-order_b2t-r2l.jpg'
+import l2r_b2t from '../../assets/images/well-order_l2r-b2t.jpg'
+import l2r_t2b from '../../assets/images/well-order_l2r-t2b.jpg'
+import r2l_b2t from '../../assets/images/well-order_r2l-b2t.jpg'
+import r2l_t2b from '../../assets/images/well-order_r2l-t2b.jpg'
+import t2b_l2r from '../../assets/images/well-order_t2b-l2r.jpg'
+import t2b_r2l from '../../assets/images/well-order_t2b-r2l.jpg'
+
+import type { WellOrderOption } from '../../form-types'
+
+interface WellOrderVisualizationProps {
+  firstValue: WellOrderOption
+  secondValue: WellOrderOption
+}
+
+const imageMap: Record<string, string> = {
+  b2t_l2r,
+  b2t_r2l,
+  l2r_b2t,
+  l2r_t2b,
+  r2l_b2t,
+  r2l_t2b,
+  t2b_l2r,
+  t2b_r2l,
+}
+
+export function WellOrderVisualization(
+  props: WellOrderVisualizationProps
+): JSX.Element {
+  const { firstValue, secondValue } = props
+  const imageKey = `${firstValue}_${secondValue}`
+
+  return (
+    <img
+      height="253px"
+      width="274px"
```

Comment:
not sure why Figma shows 253px but seems that we need to update width to align with the design.

```suggestion
      height="274px"
      width="auto" or width="100%"
```

<img width="291" alt="Screenshot 2024-10-28 at 10 05 40 AM" src="https://github.com/user-attachments/assets/a7e67a45-356c-486b-951e-1cb575c2a6ee">